tests/ecmascript/test-bi-object-assign.js

```diff
@@ -0,0 +1,147 @@
+/*
+ *  Object.assign() (E7 19.1.2.1)
+ *  http://www.ecma-international.org/ecma-262/7.0/#sec-object.assign
+ */
+
+/*===
+basic tests
+true
+maggie Kittycow Machel Fatty Whale 812 1208 9001
+0,1,2,pig,cow,ape,whale
+maggie 1208 9001 Vegeta
+cow,pig,ape,saiyan
+undefined
+this belongs here
+undefined
+prop1
+get
+set 812
+===*/
+
+function basicTest() {
+    var obj, src, dest;
+
+    // If no coercion is needed, target object is returned (i.e. not a copy).
+    obj = {};
+    dest = Object.assign(obj, { test: "test" });
+    print(dest === obj);
+
+    // Properties are assigned in [[OwnPropertyKeys]] order, moving from left
+    // to right in the argument list:
+    //     1. Array indices (sequential, ascending)
+    //     2. String keys (creation order)
+    //     3. Symbol keys (creation order)
+    // This behavior is easy to test for by assigning from multiple sources to
+    // an empty object and then calling Reflect.ownKeys() on the result.
+    obj = Object.assign({},
+        { pig: "maggie", cow: "Kittycow", ape: "Machel" },
+        [ 812, 1208, 9001 ],
+        { whale: "Fatty Whale" });
+    print(obj.pig, obj.cow, obj.ape, obj.whale, obj[0], obj[1], obj[2]);
+    print(Reflect.ownKeys(obj));
+
+    // Existing properties in the target object can be overwritten.
+    obj = Object.assign({ cow: 1208, pig: 812, ape: 9001 },
+        { pig: "maggie", saiyan: "Vegeta" });
+    print(obj.pig, obj.cow, obj.ape, obj.saiyan);
+    print(Reflect.ownKeys(obj));
```

Comment:
Hm, this comment is still showing as unfixed.  I did make the suggested change, but I guess GitHub doesn't realize it because I didn't modify this particular line.